resources/build/build-utils.sh

```diff
@@ -74,24 +74,31 @@ function findVersion() {
         VERSION_TAG=
     fi
 
-    if [ -z "${TEAMCITY_VERSION-}" ]; then
-        # Local dev machine, not TeamCity
+    if [ -z "${TEAMCITY_VERSION-}" ] && [ -z "${GITHUB_ACTIONS-}" ]; then
+        # Local dev machine, not TeamCity or GitHub Action
         VERSION_TAG="$VERSION_TAG-local"
         VERSION_ENVIRONMENT=local
-    else
+    elif [ -n "${TEAMCITY_PR_NUMBER-}" ]; then
```

Comment:
Why change from -z to -n?

Reply:
Seems easier than the double negation `! -z`. https://www.shellcheck.net/wiki/SC2236

Reply:
Gotcha, makes sense, I wasn't sure if support was the same for both `-n` and `-z`. Apparently `-n` has a little bit of a gotcha around quotation marks though: https://unix.stackexchange.com/questions/109625/shell-scripting-z-and-n-options-with-if#comment1152278_109631